app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -45,6 +47,24 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       bindToggleForSubsections();
       bindToggleOpenCloseAllButton();
 
+      // When navigating back in browser history to the accordion, the browser will try to be "clever" and return
```

Comment:
(I think the "real" solution to this is to keep the state perfectly in sync with the address bar, with something like a query string. This tends to give the most predictable and consistent behaviour. However this got vetoed by Alex, because it would make the URL ugly)